src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(x = '') {
+    if (x.length === 0) {
```

Comment:
```suggestion
  return function adder(x = '') {
    if (x.length === 0) {
```
why default `''` and check for `x.length`